### DISCUSSION TITLE
Possible to start barrier before logged in on MacOS and Windows?

### DISCUSSION THREAD

**Discussion by jjmmfi** [debauchee/barrier#1620] [Q&A]
Is there any way to start Barrier when system starts before logging in? When using a remote desktop app like Teamviewer/Chrome Remote Desktop you can remotely log in to your machine. Is that possible with Barrier?

I am working with AV production systems and running a Mac and a PC desktop on a rack in another room because they make too much noise in my studio.

**Comment by kexoyi** [comment]
Im trying to achieve this right now on MacOS. Check out my discussion showing what I have done so far: https://github.com/debauchee/barrier/discussions/1615

If you have knowledge on [launchd](https://www.launchd.info/) I have put my .plists up there for other's to test. I currently have it somewhat working where Barrier is launched on startup. Issue is, I get an error message on the server side, the DINF error on the discussion.

**Reply by pilgrimage80** [reply]
hello, can you share the plists you have prepared for macos? I want to try Macos Monterey.
thanks

**Reply by pilgrimage80** [reply]
Also, how exactly will I use it? Where do I need to copy it for Barrier to work without Login on Macos?

**Reply by pilgrimage80** [reply]
in my environment Server Windows I already have Physical keyboard and mouse. Mac OS No physical keyboard and mouse to connect to Windows computer via Macos barrier.
